### PR TITLE
876324 Github sample for Flatten annotations and form fields feature

### PR DESCRIPTION
**Pull request by VijayadharshiniMathiyalagan**
Description:
Added github sample for flatten annotations and form fields feature

### CODE REVIEW

File: Flatten/Flatten.sln [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿
```
Comment (Deepak1799) [inline Flatten/Flatten.sln:1]: Create a folder named "Save" and bring the sample inside the folder

**Save** 
* Flatten

File: Flatten/Flatten/MainPage.xaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:pv="clr-namespace:Syncfusion.Maui.PdfViewer;assembly=Syncfusion.Maui.PdfViewer"
+             x:Class="Flatten.MainPage">
+
+    <Grid RowDefinitions="50,*">
+        <pv:SfPdfViewer x:Name="pdfViewer" Grid.Row="1"/>
+        <HorizontalStackLayout HorizontalOptions="Start">
+            <Button Text="&#xe75f;"
```
Comment (Deepak1799) [inline Flatten/Flatten/MainPage.xaml:10]: image

Please name the events based on purpose and also as we discussed, modify the UI

File: Flatten/Flatten/MainPage.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        Stream stream = new MemoryStream();
+        pdfViewer.SaveDocument(stream);
+        string fileName = "SavedPDF.pdf";
+#if WINDOWS
+       string filePath = System.IO.Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.Desktop), fileName);
+       using (FileStream fileStream = new FileStream(filePath, FileMode.Create))
+       {
+           stream.CopyTo(fileStream);
+       }
+#elif ANDROID || IOS || MACCATALYST
```
Comment (Deepak1799) [inline Flatten/Flatten/MainPage.xaml.cs:46]: #elif ANDROID || IOS || MACCATALYST

Can this be **else**?

**Review by Deepak1799 — CHANGES_REQUESTED** [review]
Submitted feedback

File: Flatten/Flatten.sln (hunk 1 above)
Comment (VijayadharshiniMathiyalagan) [reply Flatten/Flatten.sln]: addressed

File: Flatten/Flatten/MainPage.xaml (hunk 2 above)
Comment (VijayadharshiniMathiyalagan) [reply Flatten/Flatten/MainPage.xaml]: addressed

File: Flatten/Flatten/MainPage.xaml.cs (hunk 3 above)
Comment (VijayadharshiniMathiyalagan) [reply Flatten/Flatten/MainPage.xaml.cs]: addressed

**Review by Deepak1799 — APPROVED** [review]
Approved

**Pull request merged**